README.md

```diff
@@ -93,6 +93,7 @@ More examples [here](https://react-semantic-ui-datepickers.now.sh).
 - size
 - transparent
 - readOnly
+- datePickerOnly
```

Comment:
This should be documented on the `Own Props` section instead, with type, description, etc.